frontend/public/components/routes.tsx

```diff
@@ -77,20 +77,95 @@ const getRouteLabel = (route) => {
+      Pending
+    </span>;
+  }
+
+  _.each(route.status.ingress, (ingress) => {
+    const isAdmitted = _.some(ingress.conditions, {type: 'Admitted', status: 'True'});
+    if (isAdmitted) {
+      atLeastOneAdmitted = true;
+    }
+  });
+
+  return atLeastOneAdmitted
+    ? <span>
+      <div className="fa fa-check text-success co-m-route-status-icon" aria-hidden="true"></div>
+      Accepted
+    </span>
+    : <span>
+      <div className="fa fa-times text-danger co-m-route-status-icon" aria-hidden="true"></div>
+      Rejected
+    </span>;
+};
+RouteStatus.displayName = 'RouteStatus';
+
+const addTLSWarnings = (route: any, warnings: string[]) => {
+  if (!route.spec || !route.spec.tls) {
+    return;
+  }
+
+  if (!route.spec.tls.termination) {
+    warnings.push('Route has a TLS configuration, but no TLS termination type is specified. TLS will not be enabled until a termination type is set.');
+  }
+
+  if (route.spec.tls.termination === 'passthrough' && route.spec.path) {
+    warnings.push(`Route path "${ route.spec.path }" will be ignored since the route uses passthrough termination.`);
+  }
+};
+
+const addIngressWarnings = (route: any, warnings: string[]) => {
+  if (!route.status) {
+    return;
+  }
+
+  _.each(route.status.ingress, (ingress) => {
+    const condition: any = _.find(ingress.conditions, { type: 'Admitted', status: 'False' });
+    if (condition) {
+      let message = `Requested host '${ ingress.host || '<unknown host>' }' was rejected by the router.`;
+      if (condition.message || condition.reason) {
+        message += ` Reason: ${ condition.message || condition.reason }.`;
+      }
+      warnings.push(message);
+    }
+  });
+};
+
+export const RouteWarnings: React.SFC<RouteHostnameProps> = ({obj: route}) => {
+  const warnings: string[] = [];
+
+  if (!route) {
+    return;
+  }
+
+  addTLSWarnings(route, warnings);
+  addIngressWarnings(route, warnings);
+  return (<React.Fragment>
+    {warnings.map((warning, index) => <div key={`routWarn-${index}`} className="co-m-route-warnings">{warning}</div>)}
+  </React.Fragment>);
+};
+RouteWarnings.displayName = 'RouteWarnings';
 
 const RouteListHeader: React.SFC<RouteHeaderProps> = props => <ListHeader>
   <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="spec.host">Hostname</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="spec.host">Location</ColHead>
   <ColHead {...props} className="col-md-2 col-sm-4 hidden-xs" sortField="spec.to.name">Service</ColHead>
-  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="spec.port.targetPort">Target Port</ColHead>
-  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="spec.tls.termination">TLS Termination</ColHead>
+  <ColHead {...props} className="col-md-4 hidden-sm hidden-xs">Status</ColHead>
```

Comment:
I think in a follow on PR we should add a filter for route status like we have for pods.

https://github.com/openshift/console/blob/master/frontend/public/components/pod.jsx#L283-L294

![screen shot 2018-05-22 at 9 01 11 am](https://user-images.githubusercontent.com/1167259/40363925-eaee08e2-5d9e-11e8-896b-16a12142a96b.png)

Reply:
Ok, created story: https://jira.coreos.com/browse/CONSOLE-486